rust 1.42.0

chenrui333 opened the pull request:
Created with `brew bump-formula-pr`.

Bo98 commented:
Cargo needs updating to 0.43.0 and the `cargobootstrap` resource needs updating to 0.42.0.

Bo98 commented:
The build will take several hours and there will be failures due to #50815. I'll cancel the current one until Cargo is updated.

chenrui333 commented:
Thanks! Yeah, I just updated the cargo version.

I was mostly thinking to create the PR with `brew bump-formula-pr` and updated the cargo from there.

Bo98 commented on Formula/rust.rb:
```diff
@@ -40,8 +40,8 @@ class Rust < Formula
-    url "https://static.rust-lang.org/dist/2019-12-19/cargo-0.41.0-x86_64-apple-darwin.tar.gz"
-    sha256 "1ef77a6be5e697bb7dde40854651fc67e91f119d5a9ddf747a25e30c1179fbe1"
+    url "https://static.rust-lang.org/dist/2020-03-12/cargo-0.43.0-x86_64-apple-darwin.tar.gz"
+    sha256 "92d4c9fb4747dce158cdfb773651aea8eac894277f3a2de5aa2c3b9d92439d8e"
```
No, this one should be 0.42.0 as per the comment.

https://github.com/rust-lang/rust/blob/1.42.0/src/stage0.txt

chenrui333 replied:
Somehow I could not find the link for `cargo-0.42.0-x86_64-apple-darwin.tar.gz`, feel free to update it.

Bo98 replied:
https://static.rust-lang.org/dist/2020-02-27/cargo-0.42.0-x86_64-apple-darwin.tar.gz

chenrui333 replied:
You are awesome!

chenrui333 replied:
Just curious, how did you find it?

I tried the date listed on the cargo github releases page, it did not quite work.

Bo98 replied:
I used the date in the stage0.txt file.

chenrui333 replied:
I see

chenrui333 commented:
# build failure
- `diesel`
- `gitg`
- `grin`
- `grin-wallet`
- `pijul`
- `rav1e`

# test failure
- `broot`

# shasum mismatch
- `gleam`

chenrui333 commented:
Give this another run.

chenrui333 commented:
gleam and diesel should be good now.

chenrui333 commented:
## catalina

diesel
efl
gitg
glade
gleam
grin
grin-wallet

## high-sierra

- diesel
- gitg
- glade
- grin
- grin-wallet

## mojave

diesel
gitg
glade
gleam
grin
grin-wallet

chenrui333 commented:
Filed Glade bug report in here, https://gitlab.gnome.org/GNOME/glade/-/issues/420

Bo98 commented:
Glade is our problem. docbook-xsl is currently broken after 1.79.2 was merged.

chenrui333 commented:
noted, do we have a fix PR for it?

Bo98 commented:
#52041
